go.mod

```diff
@@ -119,3 +121,5 @@ replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+inc
 replace google.golang.org/grpc => google.golang.org/grpc v1.26.0
 
 replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190718183610-8e956561bbf5
+
+replace github.com/dexidp/dex => github.com/pachyderm/dex v0.0.0-20201118163231-aa20083dfa7f
```

Comment:
For now we have a fork that has one change - it supports starting with no IDP configured, and adding one later. We can probably upstream this or work around it another way in the future to avoid maintaining a fork.